setup.cfg

```diff
@@ -57,4 +62,21 @@ line_length = 127
 lines_between_types = 1
 lines_after_imports = 2
 known_first_part = build
-known_third_party = nox,pallets_sphinx_themes,pep517,pytest,setuptools,toml
+known_third_party = pallets_sphinx_themes,pep517,pytest,setuptools,toml
+
+[coverage:paths]
+source =
+    src
+    .tox/*/lib/python*/site-packages
+    .tox/pypy*/site-packages
+    .tox\*\Lib\site-packages\
+    */src
+    *\src
```

Comment:
Because we run coverage on what's actually used (e.g. installed package, and we need to map it back to the source for reporting), see https://coverage.readthedocs.io/en/coverage-5.1/config.html#paths

Reply:
Shouldn't `src` be enough? In my report I only have `src`.

Reply:
You're seeing src exactly because this source path mapping is in place. Remove the other lines and would not be.